### PR TITLE
Allows :ju and :jumps command to work. 

### PR DESCRIPTION
**Pull request by zimio**
**What this PR does / why we need it**:

Allows the parsing of the jump command. This opens a small dialog where the user can select which place to jump to.

**Which issue(s) this PR fixes**

None that I'm aware of.

### CODE REVIEW

File: src/cmd_line/subparser.ts [hunk 1]
```diff
@@ -83,7 +82,11 @@ export const commandParsers = {
 
   jumps: {
     abbrev: 'ju',
-    parser: parseJumpsCommandArgs,
+    parser: () => new JumpsCommand(),
+  },
+
+  jump: {
```
Comment (J-Fields) [inline src/cmd_line/subparser.ts:88]: This shouldn't be necessary - the `abbrev` above should make everything between `ju` and `jumps` parse to `JumpsCommand`

Comment (zimio) [reply src/cmd_line/subparser.ts:88]: Vim handles three aliases: ju, jump and jumps for the same command. In order to follow vim as close as possible I added 'jump'.

Comment (J-Fields) [reply src/cmd_line/subparser.ts:88]: You'll find that `jum` works as well 🙂 Using `abbrev` makes everything between `abbrev` and the full name work, so without this entry into the map, `jump` will work (as will `jum`)

Comment (zimio) [reply src/cmd_line/subparser.ts]: Oh, I didn't know that. That's pretty great. I have removed it now.

File: src/cmd_line/subparser.ts [hunk 2]
```diff
@@ -79,6 +80,15 @@ export const commandParsers = {
     parser: parseHistoryCommandArgs,
   },
 
+  jumps: {
+    abbrev: 'ju',
+    parser: () => new JumpsCommand(),
+  },
+
+  clearjumps: {
```
Comment (J-Fields) [inline src/cmd_line/subparser.ts:88]: This should have an `abbrev` of `cle`

File: src/cmd_line/commands/jumps.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+import { VimState } from '../../state/vimState';
+import { globalState } from '../../state/globalState';
+import { Jump } from '../../jumps/jump';
+import { Range } from '../../common/motion/range';
+
+class JumpPickItem implements QuickPickItem {
+  jump: Jump;
+
+  label: string;
+  description?: string | undefined;
```
Comment (J-Fields) [inline src/cmd_line/commands/jumps.ts:13]: These `| undefined`s are unnecessary because you've got the `?`s

File: src/cmd_line/subparser.ts (hunk 2 above)
Comment (zimio) [reply src/cmd_line/subparser.ts:88]: That is true. I have added it.

File: src/cmd_line/commands/jumps.ts (hunk 3 above)
Comment (zimio) [reply src/cmd_line/commands/jumps.ts]: I have remove them.

**Pull request merged**